.circleci/config.yml

```diff
@@ -67,6 +67,28 @@ jobs:
             elif ! (curl https://api.github.com/repos/pandas-dev/pandas/issues/$CIRCLE_PR_NUMBER | jq '.labels' | grep -q 'Build'); then
                 circleci-agent step halt
             fi
+      - run:
+          # The process here is more complicated than on Github Actions
+          # since we need to copy the tests wheel into the unzipped sdist
+          # so that it gets copied (along with the contents of the unzipped sdist)
+          # into the Linux docker image used for building the wheel
+          # (unlike Github Actions there is no host access using cibuildwheel with CircleCI)
+          name: Build the sdist
+          command: |
+            pip3 install build setuptools-scm wheel
+            python3 -m build --sdist
+            sdist_name=$(ls ./dist/*.tar.gz)
+            mkdir unzipped-sdist
+            tar -xzf $sdist_name -C unzipped-sdist --strip-components=1
+            cd pandas
+            # Have to disable isolation otherwise
+            # we won't be able to pull the version correctly
+            python3 -m build --wheel --no-isolation --outdir ../unzipped-sdist
+            cd ..
+            echo "sdist_name=./unzipped-sdist" >> "$BASH_ENV"

```

Comment:
Is $BASH_ENV a circleci thing? Does it work with other shells?

Reply:
I think so. Each CI provider seems to have their own thing for setting env vars...